game/scripts/vscripts/Tutorial/Core.ts

```diff
@@ -11,6 +14,8 @@ export abstract class Section {
 
     }
 
+    public abstract sectionState: SectionState
```

Comment:
sectionState seems more like a variable used for storing the state while it is running, how about `prerequisiteState` or `initialState`?

Reply:
Named it RequiredState.